inference-engine/src/transformations/src/transformations/convert_precision.cpp

```diff
@@ -10,11 +10,14 @@
 #include <ngraph/opsets/opset4.hpp>
 #include <ngraph/opsets/opset3.hpp>
 #include <ngraph/opsets/opset1.hpp>
+#include <ngraph_ops/type_relaxed.hpp>
+#include <ngraph/pass/visualize_tree.hpp>
```

Comment:
nit: redundant header

Reply:
Yes :) If @slyalin won't left some important comments I'll remove it later.

Reply:
done